utils/src/lib.rs

```diff
@@ -269,6 +269,7 @@ impl AsyncDropFutures {
 
 /// Container for number of bytes.
 #[derive(
+    Add,
```

Comment:
Can you add `Sub` `AddAssign` and `SubAssign` also?

Reply:
sure thing

Reply:
I have added `AddAssign`. 
`Sub` and `SubAssign` derive's did not work and had to implemented manually. Since we are not using these, i thought it does not worth adding more code. But if you feel like they should be implemented, let me know, I can implement them manually